### DISCUSSION TITLE
Custom events from gltf animation

### DISCUSSION THREAD

**Discussion by goldenratio** [jnsmalm/pixi3d#151] [Q&A] [answered]
Hey,
I have general question regarding gltf. 
is it possible to trigger events from gtlf animation, similar to spine http://en.esotericsoftware.com/spine-events.

I tried googling around didn't find a solution. Thanks for your help, much appreciated!

**Comment by jnsmalm** [comment, marked as answer]
Hey,

This guy have done exactly this at https://github.com/jnsmalm/pixi3d/discussions/108

**Reply by goldenratio** [reply]
awesome!! thanks :D I should have read the discussion board better
